<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorLogDialog</class>
 <widget class="QDialog" name="ErrorLogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Error Log Found</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="icon"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;b&gt;An error log file was found. What should be done with it?&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="logEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="emailButton">
       <property name="toolTip">
        <string>Press to send an email</string>
       </property>
       <property name="text">
        <string>Send Bug Email</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="toolTip">
        <string>Press to ignore the error and delete the log file</string>
       </property>
       <property name="text">
        <string>Ignore and Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="keepButton">
       <property name="toolTip">
        <string>Press to ignore the error but keep the log file</string>
       </property>
       <property name="text">
        <string>Ignore but Keep</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>logEdit</tabstop>
  <tabstop>emailButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>keepButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
